web-docs/concepts/index.md

```diff
@@ -0,0 +1,190 @@
+# flagd basics
+
+Your flagd journey will start by defining your feature flags.
+
+flagd will then read those feature flags and make them available to your application.
+
+Your application will interact with flagd via the [OpenFeature SDK](https://example.com) to retrieve flag values via the flagd API.
+
+![flagd architecture](../images/flagd-logical-architecture.jpg)
+
+## Defining feature flags
+
+Flags can be defined in either [JSON](https://github.com/open-feature/flagd/blob/main/samples/example_flags.flagd.json) or [YAML](https://github.com/open-feature/flagd/blob/main/samples/example_flags.flagd.yaml) syntax and the values can be of different types.
+
+Here are two flags, `flagOne` has `boolean` values and `flagTwo` has `string` values.
+
+### flags represented as JSON
+
+```json
+{
+  "flags": {
+    "flagOne": {
+      "state": "ENABLED",
+      "variants": {
+        "on": true,
+        "off": false
+      },
+      "defaultVariant": "on",
+      "targeting": {}
+    },
+    "flagTwo": {
+      "state": "ENABLED",
+      "variants": {
+        "key1": "val1",
+        "key2": "val2"
+      },
+      "defaultVariant": "key1",
+      "targeting": {}
+    }
+  }
+}
+```
+
+### flags represented as YAML
+
+```yaml
+flags:
+  flagOne:
+    state: ENABLED
+    variants:
+      'on': true
+      'off': false
+    defaultVariant: 'on'
+    targeting:
+  flagTwo:
+    state: ENABLED
+    variants:
+      key1: val1
+      key2: val2
+    defaultVariant: 'key1'
+    targeting:
+```
+
+## The structure of a flag
+
+Each flag has:
+
+- A flag key: `flagOne` and `flagTwo` above
+- A state: `ENABLED` or `DISABLED`
+- One or more possible `variants`. These are the possible values that a flag key can take.
+- An optional `targeting` rule (explained below)
+
+## Targeting rules
+
+Imagine you are introducing a new feature. You create a flag with two possible variants: `on` and `off`. You want to safely roll out the feature.
+Therefore the flags `defaultValue` is set to `off` for all users.
+
+In other words, the new feature is disabled by default.
+
+Now imagine you want to enable the feature, but only when both of these conditions are true:
+
+- Logged in users
```

Comment:
The targeting rule does not cover this scenario and only validates the email sent in the context

IMO, we can reword the example to give the idea of "enabling new feature to logged in users who have their emails ending with a specific domain - @example.com"

Reply:
Technically, the user would need to be logged for the email matching to work but I agree the phrasing makes it sound like the targeting rule is checking both criteria specifically.